### PR TITLE
feat: define train function / add train loop

### PR DESCRIPTION
**Pull request by Junyoung1225**
### 변경내용
---

**define train function : 학습에 사용할 기본 상태 세팅**
- `set_seed(args.seed)` : 랜덤 시드를 설정해서 학습할 때 동일한 값이 나오도록 고정하였습니다.
-  `device` : cuda(GPU)사용이 가능하면 cuda를, 불가능하면 cpu를 사용하도록 설정하였습니다.
-  `model = LeNet5(num_classes=10)`  : 손글씨 인식에 뛰어난 LeNet5 모델로 초기화하고 mnist dataset을 사용할 것이기 때문에 num_class는 10으로 하였습니다.
- `optimizer` : 빠르고 안정적인 학습을 위해 Adam 옵티마이저로 설정하였습니다.
- `criterion` : `CrossEntropyLoss()` 다중 분류 모델이기 때문에 교차 엔트로피 손실 함수로 설정하였습니다. 

**add train loop : 기본 훈련 루프 기능 추가**
- `for epoch in range(args.epochs)`  : 에포크 수를 입력 받고 그 수만큼 반복해서 학습하게 하였습니다.
- `model.train` : 모델을 학습 모드로 설정합니다.
- `for batch_in, batch_out in tqdm(train_loader)` : 학습 데이터 로더에서 배치를 가져와 반복하게끔 하였습니다.
- `loss.backward()` : 역전파를 통하여 gradient를 계산하도록 하였습니다.
- `optimizer.step()` : 학습한 것을 바탕으로 optimizer가 parameter를 업데이트 하도록 하였습니다.
- `epoch_loss`,`epoch_acc` : 학습을 할 때마다 각 에포크의 손실율과 정확도를 계산합니다.
- 입력받은 epoch 횟수만큼 훈련한 후, 손실율과 정확도를 출력하도록 하였습니다.

### CODE REVIEW

**Comment by yongwookim1** [comment]
LGTM
